### ISSUE TITLE
Ambigous meaning for parameter `fileName` in `introprog.IO`

### ISSUE THREAD

**Issue by wudse20** [lunduniversity/introprog-scalalib#14]
The documentation in the file `introprog.IO` for the parameter `fileName` is a bit ambiguous. After trying to use this library myself and after reading in the course forum in the channel `frågor-och-svar`, I’ve come to the conclusion that it probably should be renamed to path or add documentation to explain that it’s the path of the file and not just the file name.

**Comment by bjornregnell** [comment]
Yes,file is probably a better param name than fileName. @jos-rosenqvist-3812 What do you think? Maybe this PR https://github.com/lunduniversity/introprog-scalalib/pull/15 with doc improvement helps to clear the ambiguity enough to make it less urgent to change the param name? After all param name changes can break existing client code and requires update of all examples etc...

**Comment by bjornregnell** [comment]
A global search on `fileName` (case sensitive, whole word) in the [introprog repo](https://github.com/lunduniversity/introprog) gives 158 ! occurences of `fileName` so this change is heavy and needs some careful reviewing.
